R/sql.reader.R

```diff
@@ -61,8 +61,11 @@
 #' dsn: sample_dsn
 #' user: sample_user
 #' password: sample_password
-#' dbname: sample_database
```

Comment:
I think on reflection it does. It will also have an impact on the user, password and dsn fields. I'll fix this.